difflint/lint.py

```diff
@@ -41,44 +41,30 @@ def run_command(self, args):
         self.output += output_bytes.decode()
 
 
-def _lint_javascript(file_to_lint, mode=None):
+def _lint_javascript(file_to_lint):
```

Comment:
Without modes is there still a way to have two (or more) different linter configurations for JavaScript files in the same repository?